Manak_Leah_ndvi_2022.py

```diff
@@ -0,0 +1,670 @@
+
+
+# # Question 1 (10 points)
+# 
+# Imagine that you are planning NEON’s upcoming flight season to capture remote sensing data in these locations and want to ensure that you fly the area when the vegetation is the most green.
+# 
+# When would you recommend the flights take place for each site? 
+# 
+# Answer the question in 2-3 sentences in the Markdown cell below.
+
+# Based on the plot, I would recommend the flights to take place in different times for the different locations. I would say April for the SJER site, and July for the HARV site.
+
+# # Question 2 (10 points)
+# 
+# How could you modify your workflow to look at vegetation changes over time in each site? 
+# 
+# Answer the question in 2-3 sentences in the Markdown cell below.
+
+# Instead of comparing the two sites, I would create a separate plot for each site. Each location plot would have a line for each year as a separate color, showing how the vegetation might change each month over a span of a certain amount of years. 
+
+# # Do not edit this cell! (10 points)
+# 
+# The notebook includes:
+# * additional Markdown cells throughout the notebook to describe: 
+#     * the data that you used - and where it is from
+#     * how data are being processing
+#     * how the code is optimized to run fast and be more concise
+
+# # Do not edit this cell! (20 points)
+# 
+# The notebook will also be checked for overall clean code requirements as specified at the **top** of this notebook. Some of these requirements include (review the top cells for more specifics): 
+# 
+# * Notebook begins at cell [1] and runs on any machine in its entirety.
+# * PEP 8 format is applied throughout (including lengths of comment and code lines).
+# * No additional code or imports in the notebook that is not needed for the workflow.
+# * Notebook is fully reproducible. This means:
+#    * reproducible paths using the os module.
+#    * data downloaded using code in the notebook.
+#    * all imports at top of notebook.
+
+# ## BONUS - Export a  .CSV File to Share (10 points possible)
+# 
+# This is optional - if you export a **.csv** file with the columns specified above: Site, Date and NDVI Value you can get an additional 10 points.
+# 
+# * FULL CREDIT: File exists in csv format and contains the columns specified.
+# We will check your github repo for this file!
+# 
+
+# In[13]:
+
+
+# CSV needs to have no nan values... drop them with .dropna()
+final_NDVI_df_csv = final_NDVI_df.dropna()
+
+# Export pandas dataframe to csv file
+final_NDVI_df_csv.to_csv(os.path.join(
+    directory_path,
+    "ndvi-automation",
+    "outputs", 
+    "ndvi_df.csv"))
+
+# Export to personal
+final_NDVI_df_csv.to_csv(os.path.join(et.io.HOME,
+                                      "earth-analytics",
+                                      "earth-analytics-python-env",
+                                      "ea-2022-04-ndvi-automation-LManak",
+                                      "ndvi_df.csv"))
+
+
+# In[ ]:
```

Comment:
@LManak overall, I think you had excellent variable naming conventions and comments so I could understand what you did. Very clear and concise code - nice work!